release/models/aft/openconfig-aft-common.yang

```diff
@@ -438,6 +457,58 @@ submodule openconfig-aft-common {
     }
   }
 
+  grouping aft-common-entry-nexthop-mplsudp-state {
+    description
+      "MPLS in UDP encapsulation applied on a IPv4 or IPv6 next-hop.";
+
+    leaf src-ip {
+      type oc-inet:ip-address;
+      description
+        "The source IP address for the MPLS in UDP encapsulation may be
+        expressed using this leaf (src-ip) or if may be derived from
+        '../../interface-ref/state/subinterface'";
+    }
+
+    leaf dst-ip {
+      type oc-inet:ip-address;
+      description
+        "Destination IP address to use for the encapsulated packet.";
+    }
+
+    leaf dscp {
+      type oc-inet:ip-address;
+      description
+        "Destination IP address to use for the encapsulated packet.";
+    }
+
+    leaf ttl {
+      type uint8;
+      description
+        "This leaf reflects the configured/default TTL value that is used in the
+         outer header during packet encapsulation. When this leaf is not set,
+         the TTL value of the inner packet is copied over as the outer packet's
+         TTL value during encapsulation.";
+    }
+
+    leaf-list pushed-mpls-label-stack {
+      type oc-mplst:mpls-label;
+      ordered-by user;
+      description
+        "The MPLS label stack imposed when forwarding packets to the
+        next-hop
+        - the stack is encoded as a leaf list where the first entry in the
```

Comment:
Erroneous newline?

Should we make a typedef for this type and its description to ensure consistency across the usages?